server/src/controller/exportsController.js

```diff
@@ -17,6 +17,7 @@ export const createExport = async (req, res) => {
       exportType,
     },
   });
+  // if donatedTo = BCF Curbside and exportType = return --> weight becomes negative???
```

Comment:
if donatedTo = BCF Curbside and exportType = return --> weight becomes negative???

also when getting inventory, return is same as regular just subtracted from the regular export type?